uniauth/views.py

```diff
@@ -93,11 +97,17 @@ def _login_success(request, user, next_url, drop_params=[]):
         suffix = ''
         if REDIRECT_FIELD_NAME in query_params:
             del query_params[REDIRECT_FIELD_NAME]
+        if api_mode:
+            # get object containing refresh and access token for user
+            jwt_tokens = get_tokens_for_user(user, auth_method=cas_auth)
+            # combine query_params object with jwt_tokens object
+            query_params.update(jwt_tokens) 
         if len(query_params) > 0:
             suffix = '?' + urlencode(query_params)
         return HttpResponseRedirect(next_url + suffix)
 
 
+@ensure_csrf_cookie
 def login(request):
     """
     Authenticates the user, then redirects them to the
```

Comment:
🤔 What do you think of this? There's enough changes required to support token auth that it could justify having it's own method / set of methods. Furthermore, I can forsee an application that could want the UI version of Uniauth, while also support JWT auth for external clients (in fact I'm working on one now 😉).

What if we added a separate set of views and URLs just for supporting JWT token auth?